.github/workflows/layers_partitions.yml

```diff
@@ -162,6 +162,12 @@ jobs:
           REMOTE_SHA=$(jq -r '.Content.CodeSha256' $layer_output)
           LOCAL_SHA=$(jq -r '.Content.CodeSha256' AWSLambdaPowertoolsTypeScriptV2.json)
           test "$REMOTE_SHA" == "$LOCAL_SHA" && echo "SHA OK: ${LOCAL_SHA}" || exit 1
+          REMOTE_DESCRIPTION=$(jq -r '.Description' $layer_output)
```

Comment:
Can we document this Verify step (and overall workflow) in a way similar to [this ](https://github.com/aws-powertools/powertools-lambda-typescript/blob/main/.github/workflows/make-release.yml) or other workflows?

This step has a lot going on and it can be hard to decipher.

Reply:
Yeah, good shout.